Initial pull request - Nightscout RTD.

andrew-warrington opened the pull request.

andrew-warrington committed:
Initial commit

andrew-warrington committed:
Updated folder structure

andrew-warrington committed:
Updated folder structure

danamlewis commented:
This is fantastic. Huge kudos, @andrew-warrington . I +1 this for merging and getting the live RTD link up, so we can get some volunteers to help finish the last few WIP pages.

andrew-warrington committed:
Updated folder structure - test

andrew-warrington committed:
Update of folder structure - test

andrew-warrington committed:
Added LICENSE and README.MD

andrew-warrington committed:
Updates to Index.rst

andrew-warrington committed:
test

andrew-warrington committed:
test

andrew-warrington committed:
update to index.rst

andrew-warrington committed:
test

andrew-warrington committed:
Update index.rst

andrew-warrington committed:
test

andrew-warrington committed:
Update index.rst

andrew-warrington committed:
update to index.rst files

andrew-warrington committed:
Update Nightscout/index.rst

andrew-warrington committed:
Update Nightscout/index.rst

andrew-warrington committed:
Update Nightscout/index.rst

andrew-warrington committed:
Update Nightscout/index.rst

andrew-warrington committed:
test

andrew-warrington committed:
Update Nightscout/EN/index.rst

andrew-warrington committed:
test

andrew-warrington committed:
test

andrew-warrington committed:
test

andrew-warrington committed:
test

andrew-warrington committed:
Update of TOC

andrew-warrington committed:
Update Nightscout/index.rst

andrew-warrington committed:
Moved images to an own subfolder

andrew-warrington committed:
Update index.rst

andrew-warrington committed:
fixed broken link

andrew-warrington commented:
Updates: 
- Added a front page linking to OpenAPS and AndroidAPS documentation as well (since this intended to be a main page for #WeAreNotWaiting documentation)
- Put all Nightscout docs in an EN subfolder to allow for translations with Crowdin
- Moved all images to their own subfolder and updated links

Apologies for the 7 billion commits. It is not overly simple to work with TOC in RTD, so I needed to iterate the build of the docs often to learn and get that right.

andrew-warrington committed:
Added CGM information

andrew-warrington committed:
typo fix
